gokart/task.py

```diff
@@ -77,7 +77,10 @@ def requires(self):
         return tasks or []  # when tasks is empty dict, then this returns empty list.
 
     def make_task_instance_dictionary(self) -> Dict[str, 'TaskOnKart']:
-        return {key: var for key, var in vars(self).items() if isinstance(var, TaskOnKart)}
+        return {key: var for key, var in vars(self).items() if isinstance(var, TaskOnKart) or self._is_list_of_task_on_kart(var)}
+
+    def _is_list_of_task_on_kart(self, x):
+        return isinstance(x, list) and x and isinstance(x[0], TaskOnKart)
```

Comment:
```suggestion
        return isinstance(x, list) and x and all([isinstance(item, int) for item in x])
```

Though it's not a big problem, it's better to check all items :)